recipes/globus-compute-common/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "globus-compute-common" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/globus-compute-common-{{ version }}.tar.gz
+    sha256: c8feec0235a323c3713840f6ef29632c301b32b0c86cc2bacc0b7b55a2859a3e
+  - url: https://raw.githubusercontent.com/funcx-faas/funcx-common/main/LICENSE
+    sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
```

Comment:
License file was recently added, so the latest stable release does not contain it. Expect it starting v0.2.1